task02/task02/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task02
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите сторону ромба");
+            var a = double.Parse(Console.ReadLine());
+            Console.WriteLine("Введите угол смежных сторон ромба(в градусах)");
+            var alpha = double.Parse(Console.ReadLine()) * Math.PI / 180;
+            var area = Math.Pow(a, 2) * Math.Sin(alpha);
```

Comment:
Квадрат лучше вычислять умножением.
```suggestion
            var area = a * a * Math.Sin(alpha);
```